go/client/chat_common.go

```diff
@@ -4,13 +4,35 @@
 package client
 
 import (
+	"fmt"
+	"strings"
+
 	"golang.org/x/net/context"
 
 	"github.com/keybase/client/go/libkb"
 	"github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/client/go/teams"
 )
 
 func CheckUserOrTeamName(ctx context.Context, g *libkb.GlobalContext, name string) (*keybase1.UserOrTeamResult, error) {
+	tlfCli, err := GetTlfClient(g)
+	if err != nil {
+		return nil, err
+	}
+	tlfQuery := keybase1.TLFQuery{
+		TlfName:          name,
+		IdentifyBehavior: keybase1.TLFIdentifyBehavior_CHAT_CLI,
+	}
+	_, err = tlfCli.CompleteAndCanonicalizePrivateTlfName(ctx, tlfQuery)
+	if err == nil {
+		ret := keybase1.UserOrTeamResult_USER
+		return &ret, nil
+	}
+	_, notFound := err.(libkb.NotFoundError)
+	if !notFound && !strings.HasSuffix(err.Error(), "is in an incorrect format") {
```

Comment:
Why do we return here? Shouldn't we just try the team anyway? A subteam might trigger this error with '.''s, but we would want to at least try to load them.